### PR TITLE
Refactor Jbang implementation

### PR DESCRIPTION
**Pull request by zakkak**
Still testing here: https://github.com/zakkak/quarkus-images/runs/1032797452

### CODE REVIEW

File: .github/build-images.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+            return -1;
+        }
+
+        // Cleanup
+        docker.prune();
+
+        return 0;
+    }
+
+    private void validateConfiguration(Configuration configuration) {
```
Comment (zakkak) [inline .github/build-images.java]: Hmm, this should be moved in Configuration. I 'll do it after the tests complete :)

**Comment by zakkak** [comment]
The results for the latest version are here: https://github.com/zakkak/quarkus-images/runs/1032873226?check_suite_focus=true

File: .github/mandrel-images.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: quarkus-mandrel.yaml
```
Comment (cescoffier) [inline .github/mandrel-images.yaml:1]: Nice!

**Review by cescoffier — CHANGES_REQUESTED** [review]
Looks great.
I've noticed that 2 things still need to be updated:

* the readme
* the pushed workflow

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Thanks @zakkak !

**Pull request merged**